extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/array/BoxedBooleanArrayExpansionKernel.java

```diff
@@ -40,15 +40,14 @@ public <A extends Any> WritableChunk<A> expand(
         final ObjectChunk<Boolean[], A> typedSource = source.asObjectChunk();
 
         long totalSize = 0;
-        for (int ii = 0; ii < typedSource.size(); ++ii) {
-            int rowLen;
-            if (fixedSizeLength > 0) {
-                rowLen = Math.abs(fixedSizeLength);
-            } else {
+        if (fixedSizeLength != 0) {
+            totalSize = source.size() * (long) fixedSizeLength;
```

Comment:
Math.abs?

Reply:
Those Math.abs previously were superfluous. Only the ListChunkWriter needs to know about the head/tail restriction.. so I had it wrong the first time around.